src/checkpointTrie.js

```diff
@@ -111,60 +118,50 @@ module.exports = class CheckpointTrie extends BaseTrie {
   createScratchReadStream (scratch) {
     const trie = this.copy()
     scratch = scratch || this._scratch
+    scratch = new DB(scratch._db)
     // Only read from the scratch
-    trie._getDBs = [scratch]
+    trie.db = scratch
     trie._scratch = scratch
     return new ScratchReadStream(trie)
   }
```

Comment:
Just wondering: is there any public use case for this method so  this is not marked as `private` in some way? Since it is taking `scratch` as a parameter?

Reply:
I kept it public only because it was public before. This could potentially be used to get modifications to the trie since checkpoint as a read stream. Not sure if there's any reason why this'd be helpful, or if anyone is using this.

Reply:
Did a quick search, doesn't seem to be used. Made it private.

Reply:
Ok, fine as well.